bip-update-process.md

```diff
@@ -0,0 +1,645 @@
+complete copy of the archive.
+
+The BIPs repository does not aim to track acceptance[^acceptance], adoption, or community consensus on BIPs except to
+provide a quick overview of BIP statuses (see [Workflow](#workflow) below) to visitors.
+
+## Motivation
+
+BIP 2 is over eight years old and was written when different concerns were pressing to the Bitcoin community. The BIP 2
+process seems to have been fashioned to facilitate design and activation of protocol changes. In the past years, BIPs
+more often describe interoperable features beyond the base protocol. We have had multiple debates about the role of
+BIP Editors, and some aspects of the process specified by BIP 2 that did not seem to achieve the intended goal. This
+proposal sunsets aspects of the BIP 2 process that did not achieve broad adoption, reduces the judgment calls assigned
+the BIP Editor role, and delineates the BIP Types more expediently.
+
+## What is a BIP?
+
+BIPs cover the range of interests of the Bitcoin community. The main topic is technology that supports the bitcoin
+currency. Most BIPs provide a concise, self-contained, technical description of one new concept, feature, or standard.
+Some BIPs describe processes, implementation guidelines, best practices, incident reports (e.g.
+[BIP 50](bip-0050.mediawiki)), or other information relevant to the Bitcoin community. However, any topics related to
+the Bitcoin protocol, Bitcoin’s P2P network, and Bitcoin client software may be acceptable.
+
+BIPs are intended to be the primary mechanism for proposing new protocol features, coordinating client standards, and
+documenting design decisions that have gone into implementations. Each BIP is primarily owned by its authors, and
+represents the authors’ opinion or recommendation. The authors are expected to foster discussion, address feedback and
+dissenting opinions, and, if applicable, advance adoption of their proposal within the Bitcoin community. BIPs may be
+submitted by anyone.
+
+### What is the significance of BIPs?
+
+BIPs do not define what Bitcoin is: individual BIPs do not represent Bitcoin community consensus or a general
+recommendation for implementation. A BIP represents a personal recommendation by the BIP author to the Bitcoin
+community. Some BIPs may never be adopted. Some BIPs may be adopted by several implementations. Some may even be
+incorporated into the Bitcoin protocol.
+
+### What is the scope of the BIPs repository?
+
+The BIPs repository is focused on information and technologies that aim to support and expand the utility of the bitcoin
+currency. Related topics that are of interest to the Bitcoin community may be acceptable. Proposals that are in direct
+contradiction to this mission, e.g. by undermining fungibility, facilitating unrelated protocols, using Bitcoin for data
+storage, or hobbling mining decentralization, may be considered out-of-scope.
```

Comment:
Given that it already says "_may_ be considered out-of-scope", I’m not sure this would amount to a significant shift in meaning.

I’ve reformulated this paragraph dozens of times. If you have strong feelings about this section, I’m open to discuss a more drastic rephrasing than that. As a litmus test, it seems to me that e.g. OpenTimestamps or colored coins would not be in scope, a payment protocol like the Lightning network should be on topic, and I can argue either way for Liquid or Taproot Assets. If you have a suggestion for objective criteria to delineate the scope, I’m all ears.